avro-data/src/main/java/io/confluent/connect/avro/AvroData.java

```diff
@@ -1044,6 +1047,18 @@ private static Object maybeWrapSchemaless(Schema schema, Object value, String ty
     return finalSchema;
   }
 
```

Comment:
Is is possible to out comments in the code regarding what is FullName and what is Name

Reply:
Under the `Names` section of the Avro spec, it says "Each has a fullname that is composed of two parts; a name and a namespace."

A fullname is just namespace + "." + name.

See https://avro.apache.org/docs/current/spec.html